atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
@alespergl You maybe be interested in: https://chromium.googlesource.com/chromium/src/+/b297db871b54cd58c32c96897a7a59af295ba796

That's included in Chromium 57.